### PR TITLE
tbi(ui5-test-writer): Support `contextPath`

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/1315

- Adds support for `contextPath` when adding ui5 OPA tests
- Adds new test to cover
- Manually tested with full app that uses `contextPath` instead of `entitySet`:

image

### CODE REVIEW

File: packages/ui5-test-writer/test/test-input/FullScreenLROPContextPath/webapp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                    }
+                },
+                "EmployeesObjectPage": {
+                    "type": "Component",
+                    "id": "EmployeesObjectPage",
+                    "name": "sap.fe.templates.ObjectPage",
+                    "options": {
+                        "settings": {
+                            "editableHeaderContent": false,
+                            "contextPath": "/Employees"
```
[the remark this answers is not in the document]
Comment (IainSAP) [reply packages/ui5-test-writer/test/test-input/FullScreenLROPContextPath/webapp/manifest.json:115]: Done thx

File: packages/ui5-test-writer/src/index.ts [hunk 2]
```diff
@@ -86,17 +86,22 @@ function createPageConfig(manifest: Manifest, targetKey: string, forcedAppID?: s
...
             appID,
             targetKey,
             componentID: target.id,
-            entitySet: target.options.settings.entitySet,
             template: SupportedPageTypes[target.name],
             isStartup: false
         };
+        if (target.options.settings.entitySet) {
+            pageConfig.entitySet = target.options.settings.entitySet;
+        } else if (target.options.settings.contextPath) {
```
Comment (schreckstefan) [inline packages/ui5-test-writer/src/index.ts:101]: I would change the order: contextPath should have precedence over entitySet (case someone has defined both)

File: packages/ui5-test-writer/templates/v4/integration/pages/ListReport.js [hunk 3]
```diff
@@ -9,8 +9,9 @@ sap.ui.define(['sap/fe/test/ListReport'], function(ListReport) {
     return new ListReport(
         {
             appId: '<%- appID %>',
-            componentId: '<%- componentID %>',
-            entitySet: '<%- entitySet %>'
+            componentId: '<%- componentID %>',<% if (locals.entitySet) { %>
+            entitySet: '<%- entitySet %>'<% } %><% if (locals.contextPath) { %>
```
Comment (schreckstefan) [inline packages/ui5-test-writer/templates/v4/integration/pages/ListReport.js:13]: if contextPath is defined, I would _not_ consider entitySet

File: packages/ui5-test-writer/src/index.ts (hunk 2 above)
Comment (IainSAP) [reply packages/ui5-test-writer/src/index.ts:101]: OK...

**Review by schreckstefan — DISMISSED** [review]

**Review by devinea — COMMENTED** [review]
Code changes and tests look good. 
I haven't tested locally.

Can you add a changeset?

**Review by devinea — APPROVED** [review]
Thanks for the update

**Review by schreckstefan — APPROVED** [review]

**Comment by IainSAP** [comment]
Thank You - @devinea @schreckstefan

**Pull request merged**